### PR TITLE
Connect redux with API

### PR DESCRIPTION
**Pull request by haroldmud**
## In this branch I did the following :point_down: 
- Added redux-thunk to my project.
- Used [fetch](https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch) or [axios](https://axios-http.com/docs/intro) for making API calls.
- Fetched book list from Bookstore API on Books page load.
- Refactored my add book and remove book features to persist my changes in the server.

### CODE REVIEW

File: src/redux/Books/Books.js [hunk 1]
```diff
@@ -1,44 +1,62 @@
...
+  // },
+  // {
+  //   id: 1,
+  //   genre: "action",
+  //   author: "Joseph Morgan",
+  //   title: "Vempire diaries",
+  //   completion: "40%",
+  //   chapter: "chapter2",
+  // },
 ];
```
Comment (Rustamxon7) [inline src/redux/Books/Books.js:24]: - [ ] Please kindly remove the unused commented code 👍🏼

File: package.json [hunk 2]
```diff
@@ -6,12 +6,14 @@
     "@testing-library/jest-dom": "^5.16.4",
     "@testing-library/react": "^13.3.0",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
```
Comment (Rustamxon7) [inline package.json:9]: - [ ] [OPTIONAL] Please also add MIT.md license file 👋🏼

**Review by Rustamxon7 — CHANGES_REQUESTED** [review]
Hi @haroldmud  , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_

emojibest_com_AnimatedSticker


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by OLIPLICHE — APPROVED** [review]
Hi @haroldmud , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**